core/templates/pages/landing-pages/topic-landing-page/topic-landing-page.component.html

```diff
@@ -92,9 +94,9 @@ <h3>Get started today! Looking to learn something else? We offer more than 10 su
     max-width: 1200px;
     padding: 4% 0;
   }
-  .oppia-landing-page-banner {
-    background: no-repeat center/cover url(<[$ctrl.backgroundBannerUrl]>) #afd2eb;
-  }
+  /* .oppia-landing-page-banner {
+    background: no-repeat center/cover url(<[backgroundBannerUrl]>) #afd2eb;
+  } */
```

Comment:
Don't leave commented out code, just mention if someone asks about why this was removed.

Reply:
Right. But for reference, I removed this and added another div taking https://stackoverflow.com/questions/27608932/ng-style-with-interpolated-value-not-rendering-the-background-image as reference, since I couldn't add interpolation to the css.

Removed the comment.